server/mqtt_test.go

```diff
@@ -3026,8 +3026,10 @@ func TestMQTTCluster(t *testing.T) {
 				for _, sn := range streams {
 					cl.waitOnStreamLeader(globalAccountName, sn)
 				}
-				cl.waitOnConsumerLeader(globalAccountName, mqttRetainedMsgsStreamName, "$MQTT_rmsgs_esFhDys3")
-				cl.waitOnConsumerLeader(globalAccountName, mqttRetainedMsgsStreamName, "$MQTT_rmsgs_z3WIzPtj")
+				// <>/<> TODO: need to find a way to wait for the consumer
```

Comment:
Can do a consumer list and match based on prefix `$MQTT_rmsgs` and suffix being our server hash.

Reply:
exactly, just wasn't sure of the easiest way in the test; will finish in the afternoon.

Reply:
I ignored the prefix, since there aren't any "other" consumers in this stream anyway